packages/app/src/resource/AuditEventPage.tsx

```diff
@@ -0,0 +1,29 @@
+import { Operator, SearchRequest } from '@medplum/core';
+import { ResourceType } from '@medplum/fhirtypes';
+import { Document, MemoizedSearchControl } from '@medplum/react';
+import React, { useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+export function AuditEventPage(): JSX.Element | null {
+  const { resourceType, id } = useParams() as { resourceType: ResourceType; id: string };
+  const navigate = useNavigate();
+  const [search, setSearch] = useState<SearchRequest>({
+    resourceType: 'AuditEvent',
+    filters: [{ code: 'entity', operator: Operator.EQUALS, value: `${resourceType}/${id}` }],
+    fields: ['id', '_lastUpdated'],
```

Comment:
Should show `outcome` & `outcomeDesc` by default. This helps debug which AuditEvents had errors